rudder-server-root/SPECS/rudder-server-root.spec

```diff
@@ -74,37 +73,48 @@ run a Rudder root server on a machine.
 # Installation
 #=================================================
 %install
-rm -rf %{buildroot}
-# Directories
-mkdir -p %{buildroot}%{rudderdir}/etc/
-mkdir -p %{buildroot}%{rudderdir}/etc/server-roles.d/
 
-install -m 644 %{SOURCE8} %{buildroot}/opt/rudder/etc/server-roles.d/
+cd %{_sourcedir}
+
+make install DESTDIR=%{buildroot}
 
 %pre
 #=================================================
 # Pre Installation
 #=================================================
 
+CFRUDDER_FIRST_INSTALL=$1
+LOG_FILE="/var/log/rudder/install/%{name}.log"
+
+echo "`date` - Starting %{name} pre installation script" >> ${LOG_FILE}
+
+## WARNING: This script is a copy of a part of the rudder-agent preinst
+
+if [ ${CFRUDDER_FIRST_INSTALL} -ne 1 ]
+then
+  mkdir -p /var/rudder/tmp
+
+  if [ -f /etc/init.d/rudder ]
+  then
+    # If old rudder service is here and enabled
+    if type chkconfig > /dev/null
+    then 
+      if chkconfig --list rudder 2>&1 | grep -q -e 3:on -e B:on
+      then
+        touch /var/rudder/tmp/migration-rudder-service-enabled-server
```

Comment:
missing the 2nd file

Reply:
We do not need it here as rudder-server-root scripts have no side effect on rudder-agent